.packit.yml

```diff
@@ -15,3 +15,10 @@ jobs:
     metadata:
       targets:
         - fedora-rawhide
+
+  # enable also smoke tests on Fedora ELN but right now it's not supported by Cruncher
+  - job: copr_build
+    trigger: pull_request
+    metadata:
+      targets:
+        - fedora-eln
```

Comment:
Great to see that this works now! However, why not just  add to the existing `targets:` list in the `job: tests`?

Reply:
Argh, I have worked with GitHub actions for too long, where the "job" name is just a human identifier. For packit it actually means something, sorry.

Reply:
I'm a noob with this, but IMO it's just a RPM build in COPR, not a (gating?) test run that builds RPMs as part of it. I think the `job: xyz` thing takes arbitrary strings but also has special values that mean more than just a name. See here: https://packit.dev/docs/configuration/

Based on that, I think it's correct.

Reply:
I already wrote the explanation here https://github.com/rhinstaller/anaconda/pull/2868#issuecomment-732145542

In short tests will always fail because cruncher (testing-farm automation) does not support ELN. So we would not get anything else than fail on that.

The difference between `tests` and `copr_build` is that `tests` will try to install the package to the resulting system which is the default test by testing farm team. However, it does not work if it is not supported by Cruncher.